arangod/Aql/MultiDependencySingleRowFetcher.cpp

```diff
@@ -387,9 +397,11 @@ void MultiDependencySingleRowFetcher::reportSkipForDependency(AqlCallStack const
       auto const reportedFullCount = _maximumSkipReport.getFullCount(reportLevel);
 
       auto const branchSkip = branchReport.getSkipped(reportLevel);
-      auto const branchFullCount = branchReport.getFullCount(reportLevel);
+      // NOTE: Unused, only used in assert which is disabled for now.
```

Comment:
I would love to enable it again, but I am afraid it will not work.
I would like to keep it for the same reason as the comments below, to make clear why it is not there.